### PR TITLE
Add multiple sites to dark-sites.config

### PR DESCRIPTION
**Pull request by Theblockbuster1**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -89,6 +89,8 @@ bondoer.fr
 bonsaihd.live
 botboy.snaz.in
 bouncingdvdlogo.com
+boxy-svg.com
```
Comment (Gusted) [inline src/config/dark-sites.config:92]: Please remove this entry 
image

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -233,6 +235,7 @@ erai-raws.info
 esportal.com
 etcroot.pw
 eternal.abimon.org
+ethermine.org
```
Comment (Gusted) [inline src/config/dark-sites.config:238]: Please remove this entry.
image

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -426,13 +430,15 @@ metropool.nl
 miaou.drycat.fr
 minehut.com
 miniroyale2.io
+mintme.com
```
Comment (Gusted) [inline src/config/dark-sites.config:433]: Please remove it, most sub-pages have light backgrounds.

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -544,6 +551,7 @@ protogen.dev
 protondb.com
 proxx.app
 psi.cynicaloptimist.me
+ptable.com
```
Comment (Gusted) [inline src/config/dark-sites.config:554]: Please remove this entry.
image

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 1 above)
Comment (Theblockbuster1) [reply src/config/dark-sites.config:92]: Oh strange, even in incognito they seem to be in dark mode by default on my end
image
and turning on the extension made practically no visible difference

Comment (Theblockbuster1) [reply src/config/dark-sites.config:92]: and also on all of the other sites

**Review by Gusted — APPROVED** [review]

**Pull request merged**